apps/remote_control/lib/lexical/remote_control/search/store.ex

```diff
@@ -175,6 +202,11 @@ defmodule Lexical.RemoteControl.Search.Store do
     {:reply, :ok, {ref, new_state}}
   end
 
+  def handle_call(message, _from, %State{} = state) do
+    Logger.emergency("Received #{inspect(message)}, but the search store isn't enabled yet.")
```

Comment:
Semantically, I think warning is sufficient here. I think of emergency as something that requires immediate intervention, but there's nothing really to be done but to wait.

Reply:
That's fair, I actually can think of a couple of ways to trigger this, so emergency is a bit much.